ts/api/wallet/wallet-api.ts

```diff
@@ -64,8 +64,8 @@ const transactions: ReadonlyArray<WalletTransaction> = [
     isoDatetime: "2018-04-17T07:34:00.000Z",
     paymentReason: "Certificato di residenza",
     recipient: "Comune di Gallarate",
-    amount: -20.02,
-    currency: "EUR",
+    amount: 20.02,
+    currency: "€",
```

Comment:
is this right? is this symbol what we get from the pagopa APIs?

Reply:
Maybe it is EUR.  I modified it with the aim of use the same currency notation on all the screens. 
Maybe it can be reviewd with the graphical updates planned the next week?